resources/popup.js

```diff
@@ -1,27 +1,29 @@
+// Use the toggle button on the popup to switch the loggingEnabled option.
+function toggle() {
```

Comment:
Okay, well, I think it's cleaner if it is inside the scope of the function. In case (probably not, but you never know) there will ever be another function with the name "toggle", there will be conflicts. Plus, as the function is only used inside the scope, why not leave it there?